### ISSUE TITLE
Remove the {{page}} macro

### ISSUE THREAD

**Issue by wbamberg** [mdn/content#3196]
This came out of https://github.com/mdn/content/pull/3008. We should remove the {{page}} macro from MDN: it's a common source of content problems.

We will need to look through the 370-odd places where it's used, and decide what to do in each case. Probably either:
* duplicate the content
* have the content in one place and just use links to it in the other places

@escattone

**Comment by escattone** [comment]
Thanks for creating this @wbamberg. It'll be nice to eventually resolve this and rid ourselves of this macro.

**Comment by hamishwillee** [comment]
Happy to help hack away at this with you (once the broad discussion of "how" we do this is resolved).

**Comment by hamishwillee** [comment]
FYI, Page macro search go me  420 hits in 351 files (of 11702 searched). Down to 385 hits in 316 files following easy fixes in #3392.

**Comment by wbamberg** [comment]
@hamishwillee , this is amazing work!

**Comment by peterbe** [comment]
Couple of my own cents here...

1. I very much support the idea of getting rid of `pages` macros. Another strong reason is that I don't even know if relative file attachments will work. It *used* to be that we used absolute URLs for images in `<img>` tags and as part of live-samples in `<pre>` tags. Now, these are all changed to local file references. Without even checking too closely, I bet it doesn't work in `page` transclusions. But I gather it's soon moot. 
2. There is a great way to find many of the broken `{{pages(...)}}` macros if you want to start with that instead of searching for all presences of it. You have to use `yari` and run `yarn dev` in one terminal, and `BUILD_FLAW_LEVELS="*:ignore, macros:warn" yarn build` in another terminal. Then simply keep watching <http://localhost:3000/en-US/_flaws?flaws=macros&search_flaws=macros%3AMacroExecutionError> and it should give you good hints for pages whose `pages` macro triggers a `MacroExecutionError`. 
2. The translation difference tool, for translators, don't point out that there's a difference in the `pages` macro. E.g. <http://localhost:3000/zh-CN/docs/Web/API/MediaElementAudioSourceNode#_flaws> doesn't point out that the `en-US` parent doesn't use `{{page(...)}}` but the `zh-CN` one does. I'll fix that.

**Comment by peterbe** [comment]
>     3\. The translation difference tool, for translators, don't point out that there's a difference in the `pages` macro

Soon fixed: https://github.com/mdn/yari/pull/4083

**Comment by peterbe** [comment]
@hamishwillee You pinged me on Slack about supplying a list of the pages where you get `MacroExecutionError`. 
I can think of a much better way :)
If I can just solve https://github.com/mdn/yari/issues/4082 you can get the same reporting using the preview server without having to run Yari directly from the `mdn/yari` repo. Then all the above steps in 2\. above would be easy to do locally over and over. So pester me to finishhttps://github.com/mdn/yari/issues/4082

**Comment by hamishwillee** [comment]
Awesome. I won't because I'm away for a couple of weeks. But when I'm back, watch out!

**Comment by wbamberg** [comment]
Do we really only have about 15 of these left?!

```
./files/en-us/web/api/rtcsessiondescription/rtcsessiondescription/index.md:        `RTCSdpType` enum; it must have one of the following values:{{page("/en-US/docs/Web/API/RTCSessionDescription", "RTCSdpType")}}
./files/en-us/web/api/rtcicetransport/state/index.md:{{page("/en-US/docs/Web/API/RTCIceTransportState", "Values")}}
./files/en-us/web/api/rtcicetransport/role/index.md:{{page("/en-US/docs/Web/API/RTCIceRole", "Values")}}
./files/en-us/web/api/rtcrtpsendparameters/encodings/index.md:{{page("/en-US/docs/Web/API/RTCRtpEncodingParameters", "Properties")}}
./files/en-us/web/api/rtcrtpcapabilities/index.md:{{page("/en-US/docs/Web/API/RTCRtpCodecCapability", "Properties")}}
./files/en-us/web/api/rtcstatsreport/index.md:{{page("/en-US/docs/Web/API/RTCStats", "Properties")}}
./files/en-us/web/api/rtcstatsreport/index.md:{{page("/en-US/docs/Web/API/RTCStatsType", "Values")}}
./files/en-us/web/api/rtcicecandidatestats/networktype/index.md:{{page("/en-US/docs/Web/API/RTCNetworkType", "Values")}}
./files/en-us/web/api/xrinputsourceevent/xrinputsourceevent/index.md:{{page("/en-US/docs/Web/API/XRInputSourceEvent", "Event types")}}
./files/en-us/web/api/htmlformelement/elements/index.md:{{page("/en-US/docs/Web/API/HTMLFormElement", "Elements that are considered form controls")}}
./files/en-us/web/api/mediatrackconstraints/facingmode/index.md:{{page("/en-US/docs/Web/API/MediaTrackSettings/facingMode", "VideoFacingModeEnum")}}
./files/en-us/web/api/rtciceserver/credentialtype/index.md:{{page("/en-US/docs/Web/API/RTCIceCredentialType", "Values")}}
./files/en-us/web/api/htmlvideoelement/videowidth/index.md:{{page("/en-US/docs/Web/API/HTMLVideoElement/videoHeight", "About intrinsic width and height", 0, 1)}}
./files/en-us/web/api/webgl_api/using_extensions/index.md:{{page("en-US/docs/Web/API/WebGL_API", "Extensions")}}
./files/en-us/web/api/paymentresponse/retry/index.md:    {{page("/en-US/docs/Web/API/PaymentValidationErrors", "Properties")}}
```

**Comment by hamishwillee** [comment]
Yes. I'm seeing 20 hits in 16 files

**Comment by queengooborg** [comment]
Fixed by #15906!

**Comment by hamishwillee** [comment]
Well done all. This was a slog.
